prow/cluster/monitoring/alertmanager-prow_secret.yaml

```diff
@@ -13,13 +13,15 @@ stringData:
     route:
```

Comment:
This file has been applied to the cluster (with secret fields populated). I confirmed that the new config has been picked up and parses without error.
It would be great to get this included as part of the postsubmit bazel deployment, but I'm not sure how to substitute the secret values and still use rules_k8s. I was imagining that we could store the secret URLs in k8s secrets that we fetch and use to populate the secret fields in this file before `kubectl apply`ing it.
@fejta @stevekuznetsov @hongkailiu What are your thoughts on the best way to automatically populate the secret fields before deploying with Bazel?